conda/cli/main_list.py

```diff
@@ -131,10 +139,11 @@ def execute(args, parser):
         format = 'export'
     else:
         format = 'human'
+
     if context.json:
         format = 'canonical'
 
-    exitcode = print_packages(prefix, regex, format, piplist=args.pip,
+    exitcode = print_packages(prefix, regex, format, args.reverse, piplist=args.pip,
```

Comment:
For consistency, let's pass this as a key value argument.

```suggestion
    exitcode = print_packages(prefix, regex, format, reverse=args.reverse, piplist=args.pip,
```